editor/editor_node.cpp

```diff
@@ -3688,6 +3697,13 @@ void EditorNode::_quick_run() {
 	_run(false, quick_run->get_selected());
 }
 
+void EditorNode::_execute_command() {
+	String selected_command = command_palette->get_selected_command();
+	ERR_FAIL_COND(selected_command.is_empty());
+	command_palette->execute_command(selected_command);
+	command_palette->set_selected_commmad("");
+}
```

Comment:
This method seems to only operate on the command palette, it does nothing in the editor. Wouldn't it be better to remove the signal and just call the `execute_command()` directly withing command palette?